src/ConfigurationException/InvalidConfigurationException.php

```diff
@@ -24,11 +24,16 @@
 class InvalidConfigurationException extends \InvalidArgumentException
 {
     /**
-     * @param string   $message
-     * @param int|null $code
+     * @param string          $message
+     * @param int|null        $code
+     * @param \Exception|null $previous
      */
-    public function __construct($message, $code = null)
+    public function __construct($message, $code = null, \Exception $previous = null)
```

Comment:
Good catch, but won't it break on PHP 5.x? If so, this exception class is only supposed to wrap `OptionsResolver`'s exceptions, so it should not be a big issue.